Extension/src/nativeStrings.json

```diff
@@ -214,6 +214,7 @@
         "text": "Returns:",
         "hint": "This label is for the return value description for a function. Usage example: 'Returns: Area of a shape.'"
     },
+    "sa_label": "See also:",
```

Comment:
"Template Parameters" is capitalized, so this one probably should be too.  "See Also:"

Reply:
I will keep it lowercase to match existing examples of doxygen documentation 
(see: https://eigen.tuxfamily.org/dox/classEigen_1_1Map.html).